etc/kayobe/ansible/cis.yml

```diff
@@ -6,12 +6,19 @@
   tasks:
     # TODO: Remove this when Red Hat FIPS policy has been updated to allow ed25519 keys.
     # https://gitlab.com/gitlab-org/gitlab/-/issues/367429#note_1840422075
-    - name: Assert that we are using a supported SSH key
-      assert:
-        that:
-          - ssh_key_type != 'ed25519'
-        fail_msg: FIPS policy does not currently support ed25519 SSH keys on RHEL family systems
-      when: ansible_facts.os_family == 'RedHat'
+    - when: ansible_facts.os_family == 'RedHat'
+      block:
+       - name: Check type of key using the file command
```

Comment:
I suppose someone could be using an agent, so maybe I can't make the assumption that this path contains the key.

Reply:
I've decided just to document this better in terraform-multinode as I don't think we could catch all cases:

https://github.com/stackhpc/terraform-kayobe-multinode/pull/57